app/views/templates/root_node.html.haml

```diff
@@ -16,77 +16,75 @@
   -#              = category.short_summary
 
   %section.government-links
-    %h2
-      Entry points
     .federal-government
-      %h3.home-heading.small Australian Government
+      %h2.home-heading.small Australian Government
       %ul.list-horizontal
         %li.ausgov
           %span.image
           %article
             %h3
-              %a{href: 'http://www.australia.gov.au/'} www.australia.gov.au
-            %p Entry point for people to find Australian Government information.
+              %a{href: 'http://www.australia.gov.au/'} australia.gov.au
+            %p Entry point for people to find Australian Government information. Also includes links to popular state, territory and local government information.
         %li.busgov
           %span.image
           %article
             %h3
-              %a{href: 'https://www.business.gov.au/'} www.business.gov.au
-            %p Helps businesses to find all the information and tools they need from government.
+              %a{href: 'https://www.business.gov.au/'} business.gov.au
+            %p Helps businesses to find all the information and tools they need from government to start, run and grow a business.
         %li.mygov
           %span.image
           %article
             %h3
               %a{href: 'https://my.gov.au/'} my.gov.au
-            %p Lets you link a range of Australian Government services, all in one place.
+            %p A secure myGov account lets you link a range of Australian Government services with one username and password, all in one place.
     .state-government
-      %h3.home-heading.small State and territory governments
+      %h2.home-heading.small State and territory governments
       %ul.list-vertical
         %li.nsw
           %span.image
           %article
             %h3.state-title New South Wales
-            %a{href: 'https://www.nsw.gov.au/'} www.nsw.gov.au
```

Comment:
I admit that www looks old and clunky but the URL is technically www.nsw.gov.au if someone doesn't follow the link and type the URL into the browser we should make sure that they all redirect.